miio/airconditioningcompanion.py

```diff
@@ -166,50 +169,104 @@ def __repr__(self) -> str:
              self.mode)
         return s
 
+    def __json__(self):
+        return self.data
+
 
 class AirConditioningCompanion(Device):
     """Main class representing Xiaomi Air Conditioning Companion."""
 
+    @command(
+        default_output=format_output(
+            "",
```

Comment:
Maybe it would make sense to use a keyword argument here, otherwise it remains unclear what does "" do.